### PR TITLE
deprecate py26 compat shims

### PR DESCRIPTION
**Pull request by jbrockmendel**

### CODE REVIEW

**Comment by josef-pkt** [comment]
This pull request fixes 2 alerts - [view on lgtm.com](https://lgtm.com/projects/g/statsmodels/statsmodels/rev/pr-2c828cbe52199da4d81de43cdbe3112f60c651c4)

**fixed alerts:**

* 2 for Module-level cyclic import

---

*Comment posted by [lgtm.com](https://lgtm.com)*

File: statsmodels/graphics/functional.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 """Module for functional boxplots."""
+from itertools import cobinations
```
Comment (bashtage) [inline statsmodels/graphics/functional.py]: Typo here

**Comment by josef-pkt** [comment]
This pull request fixes 2 alerts - [view on lgtm.com](https://lgtm.com/projects/g/statsmodels/statsmodels/rev/pr-310547485bc9b31dba6f42e10c92a0921866d602)

**fixed alerts:**

* 2 for Module-level cyclic import

---

*Comment posted by [lgtm.com](https://lgtm.com)*

**Comment by bashtage** [comment]
@josef-pkt LGTM. PY26 can DIAF.

**Pull request closed without merging**